.github/workflows/clang-tidy.yml

```
@@ -40,7 +40,7 @@ jobs:
           paths: '[ "**.cpp", "**.h", "**.c", "**/CMakeLists.txt", "**/Makefile", "**.hpp", "**.cmake", "build-scripts/clang-tidy.sh", "build-scripts/clang-tidy-wrapper.sh", "build-scripts/get_affected_files.py", ".github/workflows/clang-tidy.yml" ]'
   build:
     needs: skip-duplicates
-    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' }}
+    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' || github.event_name == 'pull_request_target' && github.event.pull_request.draft == false }}
```

Got my boolean logic reversed
```suggestion
    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' && github.event_name != 'pull_request_target' || github.event.pull_request.draft == false }}
```